src/status_im/chat/handlers.cljs

```diff
@@ -223,12 +221,23 @@
     db
     (assoc db :loaded-chats (chats/get-all))))
 
-;TODO: check if its new account / signup status / create console chat
 (register-handler :initialize-chats
   [(after #(dispatch [:load-unviewed-messages!]))
    (after #(dispatch [:load-default-contacts!]))]
   ((enrich initialize-chats) load-chats!))
 
+(register-handler :reload-chats
+  [{:keys [chats] :as db} _]
```

Comment:
This PR is not finished yet :) It's marked as "in progress", and has one known issue.